File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.py
n = int(input())
num = 666   # 해당 작품의 번호
cnt = 0     # 몇번째 작품인지
while 1:
    if '666' in str(num):
        cnt += 1
    if cnt == n:    # 작품 번째가 맞으면 출력
        print(num)
        break
    num += 1

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int cnt = 0;
		int i = 666;
		while (true) {
			if (String.valueOf(i).contains("666")) {
				cnt++;
			}
			if (cnt == N) {
				break;
			}
			i++;
		}
		System.out.println(i);
	}
}